src/web/components/Callout/Radios.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+
+import { RadioGroup, Radio } from '@guardian/src-radio';
+import { FieldLabel } from './FieldLabel';
+
+type fieldProp = {
+    formField: CampaignFieldRadio;
+    formData: { [key in string]: any };
+    setFormData: React.Dispatch<React.SetStateAction<{ [x: string]: any }>>;
+};
+
+// We encapsulate the Component's state here instead of directly sourcing it from
+// `formData` to avoid unnessesary rerenders, as this was causing selection animation
+// to be applied whenever formData was changin
```

Comment:
I wonder if you couldn't fix the rerender problem with use of composition and useEffect?

Reply:
I am curious what you have in mind?
I spent some time trying to get a simpler solution working but to not much luck.

Do you think it may be preferable to instead of passing down `formData` to pass down the just the field value and lift the state to the parent to avoid a re-render?